### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by yangyulan0217** [runzhouge/MAC#5]
hello!Why did my loss is quite low when I start training,and in the end the loss didn't change too much?
like:
Step 0: loss = 0.46 (1.248 sec)
Step 5: loss = 0.47 (0.273 sec)
Step 10: loss = 0.48 (0.405 sec)
Step 15: loss = 0.48 (0.325 sec)
Step 20: loss = 0.50 (0.372 sec)
Step 25: loss = 0.50 (0.297 sec)
Step 30: loss = 0.54 (0.295 sec)
Step 35: loss = 0.53 (0.320 sec)
Step 40: loss = 0.55 (0.313 sec)
Step 45: loss = 0.52 (0.289 sec)
Step 50: loss = 0.49 (0.303 sec)
...but my test results are similar to those in the paper......

**Comment by jasonmusespresso** [comment]
Hmmm.  The loss is going down in later steps. Try training for more steps.

**Comment by yangyulan0217** [comment]
thanks for your reply,but my loss is still high after 20000 steps,just going down a little,and I found my regression loss fluctuating all the time,which causes the whole loss does't change too much.is it normal?
